collector/src/main.rs

```diff
@@ -640,7 +640,7 @@ fn cg_diff(cgout1: &Path, cgout2: &Path, path: &Path) -> anyhow::Result<()> {
         anyhow::bail!("`cg_diff` not installed.");
     }
     let output = Command::new("cg_diff")
-        .arg("--mod-filename=s#/rustc/[^/]*/##")
```

Comment:
I'm failing to remember how this original regex works, I can't remember the synatx. What does it do?

Reply:
I don't know. When you get rustc from CI, the filenames are all `???` so this may just be useless nowadays